master/internal/db/postgres_experiments.go

```diff
@@ -0,0 +1,47 @@
+package db
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+)
+
+// ExperimentLabels returns a flattened and deduplicated list of all the
+// labels in use across all experiments.
+func (db *PgDB) ExperimentLabels() (labels []string, err error) {
```

Comment:
suggestion: we've been doing the sorting and filtering directly in the API handler in api_x.go files. wdy think about keeping `db/*.go` about just receiving data from the database? maybe this would return `labelUsage` and the caller sorts it, potentially in the future based on a query param.

Reply:
Good suggestion: changed to this in the latest patch.